working-with-fhir-igs/hl7v2-to-dk-medcom/service.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/http;
+import ballerina/io;
+import ballerina/log;
+import ballerinax/health.fhir.r4;
+import ballerinax/health.hl7v2;
+import ballerinax/health.hl7v2.utils.v2tofhirr4;
+
+type Hl7Message readonly & record {|
+    string content;
+|};
+
+configurable int servicePort = 9090;
+
+function init(){
+    log:printInfo("HL7v2 to Danish Transformation Service Started..");
+}
+
+service /hl7 on new http:Listener(servicePort) {
+    resource function post parse/danish(@http:Payload string payload) returns json|string|error {
```

Comment:
/transform

Reply:
fixed in [4a378ec](https://github.com/ballerina-guides/healthcare-samples/pull/38/commits/4a378ec4f64c338c69ab412bcd053c7d95901093)